components/Event.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import { View, Text, Image, StyleSheet, TouchableOpacity } from "react-native"
+import { connect } from 'react-redux'
 
 const Event = props => {
     return (
-        <TouchableOpacity style={styles.event}>
-            <View style={styles.event__photo}>
-                <Text>PHOTO GOES HERE</Text>
+        <TouchableOpacity style={styles.event} onPress={() => props.showEvent(props.event)}>
+            <View style={styles.photo__container}>
+                {props.event.name === "Quidditch After Party" && <Image source={require("../assets/quidditch.jpg")} style={styles.event__photo} />}
```

Comment:
I think it would be better to assign a default photo based on tags, instead of event name :)

Reply:
I think that you are definitely right!
I hard coded the event name here to simulate a user uploaded photo, but in the end we'll definitely want to generate default photos based on event tags. ^^